add a doc about the URL scheme (#353)
BohemianCoding/SketchAPI: docs/guides/third-party-apps-integrations.md

---
title: 3rd Party Apps Integrations
permalink: /guides/third-party-apps-integrations/
summary: Information about how to integrate Sketch into a 3rd party app
order: 610
---

> A Sketch plugin is not the only you can interact with Sketch. You might build a new app and want to integrate with Sketch. Here are a couple of ways to have a seamless workflow between your app and Sketch.

Reviewer: ```suggestion
A Sketch plugin is not the only way you can interact with Sketch. You might have built a new app and want to integrate it with Sketch. Here are a couple of ways to have a seamless workflow between your app and Sketch.
```

## Using the JSON file format

> A Sketch file is just a zip file containing a bunch of JSON files. Your app can unzip it, modify some of the files and zip it back. You don't even need Sketch at all! You can also create a Sketch file from scratch. The [`sketch-file`](https://github.com/mathieudutour/sketch-file) library might help with that.

Reviewer: ```suggestion
A Sketch file is just a zip file containing a bunch of JSON files. Your app can unzip it, modify some of the files and zip it back. You don't even need Sketch at all! You can even create a Sketch file from scratch. The [`sketch-file`](https://github.com/mathieudutour/sketch-file) library might help with that.
```

## Using SketchTool

> SketchTool is the perfect tool to interact with Sketch. You can export previews, get metadata from a file and even run a plugin programmatically!

Reviewer: ```suggestion
SketchTool is the perfect tool to interact with Sketch. You can export document previews, get metadata from a file and even run a plugin programmatically!
```

The plugin you can run with SketchTool does not need to be installed by the user, it can be bundled in your app and executed from there. You can also install the plugin by copying to the plugin folder (`~/Library/Application Support/com.bohemiancoding.sketch3/Plugins`) if necessary.

## Using the `sketch://` URL scheme

> Sketch listens to the `sketch://` URL scheme and use can make use of that to open it with a specific sketch file for example:

Reviewer: ```suggestion
Sketch listens to the `sketch://` URL scheme and you can use it to open a specific document. For example:
```
